README.md

```diff
@@ -36,6 +36,15 @@ For more information about features go to [our Swift documentation](https://meil
 
 ## Get started
 
+### Cocoapods
+
+**MeiliSearch-Swift** is available through [CocoaPods](https://cocoapods.org). To install
+it, simply add the following line to your Podfile:
+
+```ruby
+pod 'MeiliSearch'
+```
+
```

Comment:
We should add the update or install command to download the package. But I'm not sure which one

[Pod-install](https://guides.cocoapods.org/using/pod-install-vs-update.html#pod-install)

> When you run pod install, it only resolves dependencies for pods that are not already listed in the Podfile.lock.

```
pod install
```
 
[Pod-update](https://guides.cocoapods.org/using/pod-install-vs-update.html#pod-update)

> If you run pod update with no pod name, CocoaPods will update every pod listed in your Podfile to the latest version possible.

```
pod update
```

Reply:
These are commands that the developer has to run to sync their local spec. This command is common among the people using `Cocoapods` and essential to create the project anyway. It's described here at `get started`: https://cocoapods.org/#get_started

Reply:
Ah yes, I get it! But every SDK shows how to download and install packages. Even if it is common among the people using Cocoapods, it is for example not common for me. We want to be accessible to swift beginners or cocoa pods beginners 😊

Reply:
Done.